docs/developer/end-to-end-tests.md

```diff
@@ -92,10 +92,10 @@ Our CI system relies on the [kubeapps/integration-tests](https://hub.docker.com/
 The `kubeapps/integration-tests` image is built using this [Makefile](../../integration/Makefile). Manually edit this file to specify the proper version tag.
 
 ```bash
+# Get the tags from https://hub.docker.com/r/kubeapps/integration-tests
 cd integration
-# edit the Makefile with the proper version tag
```

Comment:
There's also another reference to manually editing the Makefile to specify the proper tag on line 92.